### ISSUE TITLE
setting box_check_update to false does not disable box update checking

### ISSUE THREAD

**Issue by roderickrandolph** [test-kitchen/kitchen-vagrant#181]
**.kitchen.yml**

```
---
driver:
  name: vagrant
  box_check_update: false
  vm_hostname: localhost

platforms:
  - name: chef/centos-6.6
```
**Expect:**
```
      Vagrant.configure("2") do |c|
        c.berkshelf.enabled = false if Vagrant.has_plugin?("vagrant-berkshelf")
        c.vm.box = "chef/centos-6.6"
        c.vm.box_check_update = "false"
        c.vm.hostname = "localhost"
        c.vm.synced_folder ".", "/vagrant", disabled: true
      end
```
**Actual:**
```
      Vagrant.configure("2") do |c|
        c.berkshelf.enabled = false if Vagrant.has_plugin?("vagrant-berkshelf")
        c.vm.box = "chef/centos-6.6"
        c.vm.hostname = "localhost"
        c.vm.synced_folder ".", "/vagrant", disabled: true
      end
```

Per the [Automatic Update Checking](http://docs.vagrantup.com/v2/boxes/versioning.html) section of the Vagrant documentation, box update checking is _**enabled**_ by default if "box_check_update" is not explicitly disabled in the Vagrantfile.

**Comment by roderickrandolph** [comment]
PR #182 submitted

**Comment by cheeseplus** [comment]
Merged @4f53d5f
